inc/namespace.php

```diff
@@ -64,3 +67,42 @@ function load_plugins() {
 function override_fileupload_maxk_option() : int {
 	return 1024 * 1024;
 }
+
+/**
+ * Disable the WP fatal error handler.
+ *
+ * This is intended mostly for non-technical users to receive error information.
+ *
+ * @return boolean
+ */
+function disable_wp_fatal_handler() : bool {
```

Comment:
This is a confusing name. Can we just use `__return_false` instead?

Reply:
Yeah, turns out that function is not available that early, as the fatal handler is loaded super early

Reply:
OK, let's rename this to `filter_wp_fatal_handler` or something; having a function called `disable_...` return `false` is just a bit weird.

Reply:
Also, let's document this

Reply:
Ok